utils/post_utils.test.jsx

```diff
@@ -971,6 +971,20 @@ describe('PostUtils.splitMessageBasedOnCaretPosition', () => {
     });
 });
 
+describe('PostUtils.splitMessageBasedOnTextSelection', () => {
+    const state = {
+        selectionStart: 5,
+        selectionEnd: 12,
+    };
+
+    const message = 'Test Replace Message';
+    it('should return an object with two strings when given context and message', () => {
```

Comment:
You could modify a bit this test to add a couple of more test cases with [`test.each`](https://jestjs.io/docs/api#testeachtablename-fn-timeout), just for completion.

Reply:
Good point. I improved the test with `test.each`